### PR TITLE
[2.7.3] Fixes for issue #219 and #779 for rc2.7.3

### PR DESCRIPTION
**Pull request by chrisknoll**
Fixes for #219 and #779 for merge into rc2.7.3 branch.

### CODE REVIEW

File: src/main/resources/resources/pathway/runPathwayAnalysis.sql [hunk 1]
```diff
@@ -200,12 +175,12 @@ SELECT
   target_count.cnt AS target_cohort_count,
   pathway_count.cnt AS pathways_count
 FROM (
-  SELECT CAST(COUNT(*) AS INT) cnt
+  SELECT COUNT_BIG(*) cnt
```
Comment (anthonysena) [inline src/main/resources/resources/pathway/runPathwayAnalysis.sql]: I think we need to keep the CAST() here based on this change for Impala: 

https://github.com/OHDSI/WebAPI/commit/687d0123e7c56079c89635a44d934b9c98163626#diff-02c181aadb1a447251b457feccfa48eaR205

Comment (chrisknoll) [reply src/main/resources/resources/pathway/runPathwayAnalysis.sql]: Ok, I'll update that commit with the change.

**Comment by anthonysena** [comment]
@chrisknoll - tested again and I'm facing an issue with a pathway analysis with > 1 target cohort in the definition. Here is the error I've received:

`There is already an object named '#combo_events' in the database.`

Likely an issue with this script: `src/main/resources/resources/pathway/runPathwayAnalysis.sql `

**Comment by chrisknoll** [comment]
Good catch @anthonysena .  I've updated the PR and force-pushed the branch (after testing with multiple T's).  Please delete your local branch before pulling down the PR branch. I will also update the main PR for master branch with this update.

**Pull request merged**